test/unit/threads.test.ts

```diff
@@ -1211,10 +1214,7 @@ describe('Threads 2.0', () => {
     });
 
     describe('Methods & Getters', () => {
-      let stubbedQueryThreads: sinon.SinonStub<
-        Parameters<StreamChat['queryThreads']>,
-        ReturnType<StreamChat['queryThreads']>
-      >;
+      let stubbedQueryThreads: sinon.SinonStub<[options?: QueryThreadsOptions], ReturnType<StreamChat['queryThreads']>>;
```

Comment:
Not sure whether this change is necessary, let's return it to the previous state. :)

Reply:
Changed it back. Not sure why this got changed. Maybe when I run eslint or prettier.